docs/transpile/index.mdx

```diff
@@ -11,15 +11,22 @@ Transpilation is the process of rewriting a given input circuit to match the top
 
 A central component of Qiskit, the transpiler is designed for modularity and extensibility. The goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by a **pass manager**, which schedules the passes and also allows passes to communicate with each other by providing a shared space. In this way, the transpiler opens up the door for research into aggressive optimization of quantum circuits.
 
-This documentation covers the tooling and workflows available to Qiskit users.  If you're using primitives and are only interested in st using the default transpilation options, look to the section [Configure error suppression for Qiskit Runtime.](../run/configure-error-suppression)
+This documentation covers the tooling and workflows available to Qiskit users.  If you're using primitives and are only interested in st using the default transpilation options, look to the section [Configure error suppression for Qiskit Runtime.](../run/configure-runtime-compilation)
 
 
-## Transformation passes
-For more advanced users, it is important to understand the various representations of quantum circuits that are used for the transpilation step along with the transformations (i.e., *passes*) that affect the efficiency of the final circuit that is sent to the backend.  This involves rewriting quantum circuits to match hardware constraints and suppressing the effects of noise.  The flow of logic for this tool chain is quite customizable and need not be linear.  One can prepare iterative loops, conditional branches, and other complex behaviors.  A good starting place when developing your own set of passes is by looking to the default sequence of transformations:
+
+## Default Transpilation
+The basic usage of the transpiler is to call the `qiskit.compiler.transpile()`](../api/qiskit/qiskit.compiler) function.  This will generate one of the present `PassManager`s based on, among other options, an `optimization_level` flag which can be set to either 0, 1, 2, or 3.  Higher levels generate more optimized circuits, at the expense of longer transpilation times.  Each of these preset `PassMangers` are composed of six basic stages:
```

Comment:
@kaelynj Would this be the right link to the API? `../api/qiskit/compiler#compilation-routines` (goes [here](https://docs.quantum-computing.ibm.com/api/qiskit/compiler#compilation-routines))

Reply:
I think so?  It'd be nice if we could point directly to the function in that long api reference page, but not sure what the status of those are.  Also I can't seem to load the api pages in my local build of the site for some reason.

Reply:
I have the same problem -- I think the local build just doesn't like it!!

Reply:
Anyone know how to figure out if this [qiskit.compiler.transpile](https://qiskit.org/documentation/stable/0.28/stubs/qiskit.compiler.transpile.html) page is deprecated, since I see it on qiskit.org but not in our API docs. @Eric-Arellano? @javabster?